openedx/adg/lms/courseware_override/helpers.py

```diff
@@ -102,3 +103,19 @@ def get_extra_course_about_context(request, course):
     }
 
     return context
+
+
+def get_user_enrolled_courses(user):
+    """
+    Get all the enrolled courses of the user
+
+    Args:
+        user (User): User to get the enrolled courses for
+
+    Returns:
+        QuerySet: All the enrolled courses (CourseOverview) of the user
+    """
+    enrolled_courses_ids = CourseEnrollment.enrollments_for_user(user).values_list(
+        'course_id', flat=True
+    )
+    return CourseOverview.objects.filter(id__in=enrolled_courses_ids)
```

Comment:
How about doing it reverse? 
`CourseOverview.objects.filter(courseenrollment__user=user, is_active=1)`
@annas-usman @aightmunam

Reply:
@asadali145 this can be done in @faizan-philu's ticket we can inform Faizan and Munam to change it there, for now we can merge.

Reply:
Will take care of it @asadali145 @annas-usman